use-cases.md

```diff
@@ -69,11 +65,7 @@ The `operation` above refers to the name the server wants to give to the Span. F
 The `span` object above can be `None` if the Tracer did not find relevant headers in the incoming request: presumably because the client did not send them. In this case the server needs to start a brand new trace.
 
 ```python
-    span = tracer.join_trace_from_text(
-        operation_name=operation,
-        context_snapshot=request.headers,
-        trace_attributes=request.headers
-    )
+    span = tracer.get_span_extractor(request.class).extract(request)
```

Comment:
There's one problem with (2) above – namely that the `StringMapExtractor` is defined in an OT implementation, and as such we run into portability problems. FWIW, my own suggestion had the same issue.

How do we consistently identify Extractors/Injectors – and, worse still, any parameters they take – without breaking the core OT interface abstraction? We could do something gross with an enum (one entry for each type of extractor+injector pair).